docs/topics/auth/default.txt

```diff
@@ -1691,7 +1691,25 @@ provides several built-in forms located in :mod:`django.contrib.auth.forms`:
 
 .. class:: UserCreationForm
 
-    A form for creating a new user.
+    A form for creating a new user
+
+    **Attributes:**
+
+    * ``username``:
+        The username sent to the form.
+
+    * ``password1``:
+        The password sent to the form.
+
+    * ``password2``:
+        The password confirmation sent to the form.
+
+    Because ``UserCreationForm`` extends ``django.forms.ModelForm`` it
```

Comment:
I kinda get it, and of course if the implementation changes, the docs need to as well. Right now, it has no docs really. It's like a toy with an box saying lights, but no way of knowing how to use them or relate that to understood concepts about the toy.